File: 20260331/ex02.js
function solution(start_num, end_num) {
    var answer = [];

    let a = start_num;
    while (a <= end_num) {
        answer.push(a);
        a++;
    }
    return answer;
}

const result = solution(3, 5);
console.log(result);

// start_num 에서 end_num까지의 숫자를
// 배열로 출력하세요

// 만약에 3,5 입력하였으면 배열 [3,4,5]반환
// 만약에 1,3 입력하였으면 배열 [1,2,3]반환

File: 개인.py/ex02.py
def solution(score):
    answer = []

    scores = []
    for i in score:
        scores.append(sum(i)/2)
    scores_1 = sorted(scores, reverse=True)
    for i in scores:
        answer.append(scores_1.index(i)+1)
    return answer

results = solution([[80, 70], [90, 50], [40, 70], [50, 80]])
print(results)